Blockzilla/SettingsViewController.swift

```diff
@@ -279,8 +284,8 @@ class SettingsViewController: UIViewController, UITableViewDataSource, UITableVi
         return .lightContent
     }
     
-    func createBiometricLoginToggle() {
-        guard biometryType.hasBiometry else { return }
+    fileprivate func createBiometricLoginToggle() -> BlockerToggle? {
```

Comment:
If createBiometricLoginToggle can silently fail to make the toggle, maybe it should be renamed ```createBiometricLoginToggleIfAvailable()``` or something similar.